assets/scripts/home.js

```diff
@@ -1,48 +1,58 @@
 // Imports
-import { getProducts } from "./fetchProducts.js";
-import { openProduct } from "./openProduct.js";
-import { addToCart } from "./addToCart.js";
+
+import { changeNumberOfUnits } from "./changeNumber.js";
+import { updateCartCount } from "./updateCartCount.js";
+import { saveProductsToLocal } from "./fetchProducts.js";
 
 // Variables
 let featuredProductsContainer = document.querySelector(
   ".featured-pro-container"
 );
+saveProductsToLocal();
```

Comment:
This says save, but it then does print? I would also expect it to take an argument of what it’s meant to save